src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -0,0 +1,119 @@
+- name: Get Keycloak tags
+  uri:
+    url: https://api.github.com/repos/keycloak/keycloak/tags
+    method: GET
+    return_content: yes
+    status_code: 200
+    body_format: json
+  register: result
+
+- name: Set keycloak_url fact
+  set_fact:
+    keycloak_url: "{{ base_url }}/{{ result.json[1].name }}/keycloak-{{ result.json[1].name }}.tar.gz"
+
+- name: Create Keycloak user/group
+  user:
+    name: keycloak
+    home: /opt/keycloak
+    system: yes
+    create_home: yes
+
+- name: Print keycloak_url
```

Comment:
Please, move this after set_fact so it is togther.

Reply:
moved